model/src/main/java/org/projectnessie/api/TreeApi.java

```diff
@@ -95,6 +95,15 @@ EntriesResponse getEntries(
       @NotNull @Valid EntriesParams params)
       throws NessieNotFoundException;
 
+  EntriesResponse getNamespaceEntries(
```

Comment:
No need to add it to `TreeApi`

Reply:
+1 to not changing legacy API in this case. The UI should be able to use v1 API, I assume.

Reply:
why? Then it isn't in the client and I can't use it in tests.

Reply:
Can't test use v1 API?

Reply:
I do not mind adding a method to the old API as much as I'm wondering why it is necessary to add it to the old API rather than to v1 API :thinking: